.drone.yml

```diff
@@ -527,6 +527,18 @@ steps:
     - cd storage
     - cargo test --release --features maindb-backend-sled
 
+
+- name: run-unit-tests-multibackend-edgekv
+  image: tezedge/tezedge-ci-builder:nightly-2021-08-04-v9.5-tezos
```

Comment:
Change this one to `tezedge/tezedge-ci-builder:nightly-2021-08-04-v10.3-tezos`, it has been upgraded a few days ago.